packages/node-modules-inspector/src/app/state/filters.ts

```diff
@@ -38,15 +38,14 @@ export const workspacePackages = computed(() => Array.from(packageData.value?.pa
 
 const debouncedSearch = useDebounce(computed(() => filters.search), 200)
 
-export const filteredPackages = computed(() => Array.from((function *() {
-  for (const pkg of packageData.value?.packages.values() || []) {
+function* packageFilterGenerator(packages, filters, searchValue) {
```

Comment:
Why do we need this refactor?

Reply:
I think we could build `filteredPackageVersionsMap` from `filteredPackages` instead of running again

Reply:
I am working on it

Reply:
yeah. you are right. Missed that 😅